enterprise/app/trends/percentile_chart.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+
+import * as format from "../../../app/format/format";
+import { ResponsiveContainer, ComposedChart, CartesianGrid, XAxis, YAxis, Line, Legend, Tooltip } from "recharts";
+
+interface Props {
+  title: string;
+  data: any[];
+  extractLabel: (datum: any) => string;
+  formatHoverLabel: (datum: any) => string;
+  extractP50: (datum: any) => number;
+  extractP75: (datum: any) => number;
+  extractP90: (datum: any) => number;
+  extractP95: (datum: any) => number;
+  extractP99: (datum: any) => number;
+}
+
+const PercentilesChartTooltip = ({
+  active,
+  payload,
+  labelFormatter,
+  extractP50,
+  extractP75,
+  extractP90,
+  extractP95,
+  extractP99,
+}: any) => {
```

Comment:
replace `any` with a stronger type here too? (can write a separate non-exported interface like `interface PercentilesChartTooltipProps` to make it cleaner.)

Reply:
Done